core/src/main/java/org/apache/iceberg/DataFiles.java

```diff
@@ -352,7 +319,9 @@ public DataFile build() {
       return new GenericDataFile(
           filePath, format, isPartitioned ? partitionData.copy() : null,
           fileSizeInBytes, new Metrics(
-              recordCount, columnSizes, valueCounts, nullValueCounts, lowerBounds, upperBounds), keyMetadata);
+          recordCount, columnSizes, valueCounts, nullValueCounts, lowerBounds, upperBounds),
```

Comment:
Nit: the arguments to `Metrics` should be indented so they don't appear to be part of the outer call.